pkg/controller/plan/adapter/vsphere/builder.go

```diff
@@ -491,6 +505,64 @@ func (r *Builder) DataVolumes(vmRef ref.Ref, secret *core.Secret, _ *core.Config
 	return
 }
 
+// Return all shareable PVCs
+func (r *Builder) ShareablePVCs() (pvcs []*core.PersistentVolumeClaim, err error) {
+	pvcsList := &core.PersistentVolumeClaimList{}
+	err = r.Destination.Client.List(
+		context.TODO(),
+		pvcsList,
+		&client.ListOptions{
+			LabelSelector: k8slabels.SelectorFromSet(map[string]string{
+				Shareable: "true",
+			}),
+		},
+	)
+	if err != nil {
+		err = liberr.Wrap(err)
+		return
+	}
+	pvcs = make([]*core.PersistentVolumeClaim, len(pvcsList.Items))
+	for i, pvc := range pvcsList.Items {
+		// loopvar
+		pvc := pvc
```

Comment:
hmmm, neetpick, `pvc := pvc` is not nice, maybe `allocatedPvc := pvc` or `copyPvc := pvc` ?